deeplearning/train.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
+import numpy as np
+import util.pixnet as pixnet
+
+train_size = 10
+image_size = [625, 1250, 3]
```

Comment:
I messed up and told you the wrong image width...it's actually 640 instead of 625.